eos/constraints/B-to-D-l-nu.yaml

```diff
@@ -132,4 +132,21 @@ B^0->D^+l^-nu::KinematicalDistribution[w]@Belle-2015A:
       0.008462011094650328, 0.01124041430207108, 0.013615071527758647, 0.015895479300835378,
       0.02293292241393046]
     dof: 9
+
```

Comment:
The metadata (which source, what year) is missing. Have a look how this is done in lines 1-7.